lib/services/database_test.go

```diff
@@ -189,3 +191,41 @@ func TestDatabaseFromRDSCluster(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, expected, actual)
 }
+
+func TestAuroraMySQLVersion(t *testing.T) {
+	require.Equal(t, "5.6.10a", auroraMySQLVersion(&rds.DBCluster{EngineVersion: aws.String("5.6.10a")}))
+	require.Equal(t, "1.22.1", auroraMySQLVersion(&rds.DBCluster{EngineVersion: aws.String("5.6.mysql_aurora.1.22.1")}))
+	require.Equal(t, "1.22.1.3", auroraMySQLVersion(&rds.DBCluster{EngineVersion: aws.String("5.6.mysql_aurora.1.22.1.3")}))
+}
+
+func TestIsRDSClusterSupported(t *testing.T) {
+	logger := logrus.WithField("test", t.Name())
```

Comment:
This wouldn't be necessary if a logger was not required for `IsRDSClusterSupported` (see other comment).